core/payment/migrations/2020-01-23-123456_init/down.sql

```diff
@@ -4,9 +4,9 @@ DROP TABLE "pay_debit_note_event";
 
 DROP TABLE "pay_event_type";
 
-DROP TABLE "pay_payment_x_invoice";
+DROP TABLE pay_agreement_payment;
```

Comment:
Sometimes you have table name in quotes, sometimes not. Please be consistent.

Reply:
OK, I'll do a separate PR.

Reply:
IMHO making a separate PR for it is an overkill xd. Just remember to change it.

Reply:
I opened a separate PR to avoid noise: https://github.com/golemfactory/yagna/pull/319